packages/blocks/src/page-block/edgeless/components/rects/edgeless-selected-rect.ts

```diff
@@ -397,10 +417,13 @@ export class EdgelessSelectedRect extends WithDisposable(LitElement) {
   private _isResizing = false;
 
   @state()
-  private _dragDirection: HandleDirection | null = null;
+  private _mode: 'resize' | 'scale' | 'rotate' = 'resize';
 
   @state()
-  private _scalePercent: string | null = null;
+  private _scalePercent: string | undefined = undefined;
```

Comment:
Guess using `private _scalePercent?: string;` could be semantically cleaner.

Reply:
@doodlewind 
Done